backend/pkg/api/hooks.go

```diff
@@ -401,3 +181,107 @@ func (*defaultHooks) CanCreateRedpandaRoles(_ context.Context) (bool, *rest.Erro
 func (*defaultHooks) CanDeleteRedpandaRoles(_ context.Context) (bool, *rest.Error) {
 	return true, nil
 }
+
```

Comment:
I wonder if it's worth putting `AuthenticationDefaultHandler` into its own file?

Reply:
I wouldn't mind. Currently I opted for the way to put all default hooks in a single file but we can split the auth ones out I guessß